doc/examples/filters/plot_unsharp_mask.py

```diff
@@ -0,0 +1,46 @@
+"""
+===============
+Unsharp masking
+===============
+
+Unsharp masking is a linear image processing technique which sharpens
```

Comment:
What does 'linear' mean in this context?

Reply:
It is linear in mathematical sense. If you ignore the rounding errors and clipping due to overflow/underflow, then:
```USM(A+B) = USM(A)+USM(B)```

```USM(A*alpha) = USM(A)*alpha```

It is basically a convolution with a special kernel.